backend/utils.js

```diff
@@ -1,5 +1,25 @@
 const moment = require('moment');
 
+//https://stackoverflow.com/questions/4340227/sort-mixed-alpha-numeric-array
+var reA = /[^a-zA-Z]/g;
+var reN = /[^0-9]/g;
+
+function sortAlphaNum (aa, bb) {
+  const a = (aa || "");
+  const b = (bb || "");
+
+  const aAlpha = a.replace(reA, "");
+  const bAlpha = b.replace(reA, "");
+
+  if (aAlpha === bAlpha) {
```

Comment:
sometimes a number comes first , e.g. 5-a-side, or numbers and letters are interleaved 'Hb5a2 Classroom'.
Perhaps stick to a normal string sort unless the pattern matches 'just letters' then 'just numbers'